.gitignore

```diff
@@ -37,6 +37,8 @@ doc/source
 
 *.swp
 
+pyccel/compilers/*.json
```

Comment:
just question, why you ignored `.json` file?

Reply:
They need to be generated on your computer so that the config values provided by `sysconfig` match the setup on your computer (os, python version, libraries used when python was compiled). I don't want anyone to, for example, accidentally commit a `.json` configured for linux, which would then break the setup for anyone on mac or windows